components/style/properties/shorthand/position.mako.rs

```diff
@@ -127,3 +127,27 @@
         }
     }
 </%helpers:shorthand>
+
+<%helpers:shorthand name="grid-gap" sub_properties="grid-row-gap grid-column-gap"
+                    spec="https://drafts.csswg.org/css-grid/#propdef-grid-gap">
+  use properties::longhands::{grid_row_gap, grid_column_gap};
+
+  pub fn parse_value(context: &ParserContext, input: &mut Parser) -> Result<Longhands, ()> {
+      let row_gap = grid_row_gap::parse(context, input)?;
+      let column_gap = grid_column_gap::parse(context, input).unwrap_or(row_gap.clone());
+
+      Ok(Longhands {
+        grid_row_gap: row_gap,
+        grid_column_gap: column_gap,
+      })
+  }
+
+  impl<'a> ToCss for LonghandsToSerialize<'a>  {
+      fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+          self.grid_row_gap.to_css(dest)?;
+          dest.write_str(" ")?;
```

Comment:
I added the change, although I've been looking around and can't get any idea what's the usecase of ToCss so I can confirm with Firefox behavior.